cloud-aws-cloudformation/src/main/resources/definitions/aws-environment-minimal-policy.json

```diff
@@ -179,7 +179,8 @@
         "kms:ListAliases",
         "ec2:ModifyInstanceAttribute",
         "ec2:CreateLaunchTemplateVersion",
-        "ec2:ModifyLaunchTemplate"
+        "ec2:ModifyLaunchTemplate",
+        "ec2:DescribeLaunchTemplateVersions"
```

Comment:
Notify customers with a bulletin, otherwise this may not function as expected (I know we discussed that it is protected by exception handling but that defeats the purpose)